CEMontology/.htaccess

```diff
@@ -0,0 +1,3 @@
+RewriteEngine On
+
+RewriteRule (.*)$ http://crimeanalytics.it/cem-reference/doc/index-en.html$1 [R=302,L,QSA]
```

Comment:
I don't think this will work like you intend it. Have you tried it locally? You are trying to convert slashes to hash anchors, so you'll probably need to do something like adding a hash:
`RewriteRule (.*)$ http://crimeanalytics.it/cem-reference/doc/index-en.html#$1 [R=302,L,QSA]` (note the `#`)